compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -110,6 +110,12 @@ int entry(int argc, char **argv)
       .default_value(false)
       .help("This will fuse BatchNorm operators of pre-activations to Convolution operator");
 
+  arser.add_argument("--remove_duplicate_transpose")
+      .nargs(0)
+      .required(false)
+      .default_value(false)
+      .help("This will remove duplicate Transpose operator which is not effect on model result");
```

Comment:
I think the help message can be more clarified as below.

```suggestion
      .help("This will fuse or remove subsequent Transpose operators");
```

Reply:
I will apply this.

Reply:
I will apply this.